cycle/docker-compose.yaml

```diff
@@ -624,6 +679,14 @@ services:
     image: ${hub}/lsst_sims:${CYCLE}
     <<: *base-lsst_sims
 
+  adam-sensors:
+    image: ${hub}/adam-sensors:${CYCLE}
+    <<: *base-adam-sensors
+
+  adam-sensors_c:
```

Comment:
Looks like this is backwards from the other specs. This one should be without the `_c` and the one above should have the `_c`.